### PR TITLE
fix metadata

### PR DESCRIPTION
**Pull request by rueshyna**
metadata updating needs to be proposed by creating proposal.

cc: @tj825

### CODE REVIEW

File: src/internal/execution.mligo [hunk 1]
```diff
@@ -44,6 +44,8 @@ let send (content : proposal_content) (storage : storage_types)
     | Add_owners s -> ([], Storage.Op.add_owners s storage)
     | Remove_owners s -> ([], Storage.Op.remove_owners s storage)
     | Adjust_effective_period i -> ([], Storage.Op.adjust_effective_period i storage)
+    | Add_or_update_metadata { key; value } -> ([], Storage.Op.update_metadata (key, (Some value), storage))
```
Comment (JulesGuesnon) [inline src/internal/execution.mligo:47]: Wouldn't `Set_metadata` be shorter and a better naming? But maybe it's not as much explicit 🤔

File: test/common/helper.mligo [hunk 2]
```diff
@@ -30,7 +30,7 @@ let init_storage (owners, threshold: address set * nat) : storage_types =
   owners           = owners;
   threshold        = threshold;
   effective_period = 172800;
-  metadata         = (Big_map.empty: (string, bytes) big_map);
+  metadata         = Big_map.literal [("", 0x01)];
```
Comment (JulesGuesnon) [inline test/common/helper.mligo:33]: Why is not empty?

Comment (rueshyna) [reply test/common/helper.mligo:33]: This is initial value of storage for running creating a contract. Usually, users will add something to metadata at this time. 
There was no way to modify it once creation is done so it was not really important thing to test.
Now, we have a way to update it. Also, as it's just a test, so I put random value there for testing.

File: src/internal/execution.mligo (hunk 1 above)
Comment (rueshyna) [reply src/internal/execution.mligo:47]: I was referring the batcher's contract for naming. I was bothering by it a little bit.

**Review by JulesGuesnon — APPROVED** [review]

**Pull request merged**